scripts/translation.sh

```diff
@@ -5,19 +5,19 @@ cd "$DIR"
 echo "make translation under $DIR\n";
 
 carton exec perl local/bin/xgettext.pl  --verbose -P perl=pl,pm -P tt2=tt,tt2  \
-    --output=messages.pot   --output-dir=../src/config/locales/   --directory=../src/templates/  --directory=bin/
+    --output=messages.pot   --output-dir=../src/translations/   --directory=../src/templates/  --directory=bin/
```

Comment:
Then `src/config/locales` folder and its contents need to be refactored to `src/translations` accordingly.

Reply:
Thanks, missed this somehow.